### DISCUSSION TITLE
Has this project been abandoned?

### DISCUSSION THREAD

**Discussion by riggyz** [fnuecke/oc2#266] [Q&A]
Hey I just found this mod after looking for 1.19 alternatives to OC. Looks like the repo is stale and PR haven't pulled - is this project still being developed?

**Comment by ranenvious** [comment]
I'm curious as well. The github repo hasn't shown any activity for over a year at this point, but the issues appear to be getting responses still and I've seen mods in the past that are definitely active but have stale github repos for some reason.

**Comment by Faeranne** [comment]
I've seen Sangar disappear for weeks at a time.  Always comes back to at least mention if things are moving forward.  Life happens.

**Comment by Un1q32** [comment]
This is a very distant memory at this point but I think I remember them mentioning they couldn't update the mod because there was some critical bug in the RISC V emulator that they had to fix first, and they couldn't find the cause of the bug.

**Reply by ranenvious** [reply]
that'd probably do it. If anything is going to cause issues for a minecraft mod, it's probably going to be trying to emulate an entire CPU with an entirely different architecture. Still a bit weird that the repo is stale though, but I suppose development is just happening either offline or privately.

**Reply by SquidDev** [reply]
>  there was some critical bug in the RISC V emulator that they had to fix first

https://github.com/fnuecke/oc2/issues/253#issuecomment-1336404935

**Reply by Un1q32** [reply]
> >  there was some critical bug in the RISC V emulator that they had to fix first
> 
> https://github.com/fnuecke/oc2/issues/253#issuecomment-1336404935

8 months ago 💀

This project is super-dead

**Reply by atirutw** [reply]
Let's hope it's just a long hiatus.

**Comment by samuelh2005** [comment]
There's https://github.com/North-Western-Development/oc2r, a 1.20.1 development of OC2.
